documentation/modules/serving/pages/knative-client.adoc

```diff
@@ -306,17 +305,15 @@ To delete a specific revision you can use the command:
 [.console-input]
 [source,bash,subs="+macros,attributes+"]
 ----
-kn revision delete greeter-7cqzq
+kn revision delete greeter-twpgf-1
 ----
 
 [.console-output]
 [source,bash]
 ----
-Revision 'greeter-7cqzq' successfully deleted in namespace 'knativetutorial'.
+Revision 'greeter-twpgf-1' successfully deleted in namespace 'knativetutorial'.
```

Comment:
There's some outdated revision names here

Reply:
I feel revision name does not matter or you feel it to be consistent with names in the page ?

Reply:
Causes confusion to have a revision that's not listed before IMO.